2021-10-06 23:52:48,143 INFO    MainThread:38942 [wandb_setup.py:_flush():69] setting env: {}
2021-10-06 23:52:48,143 INFO    MainThread:38942 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-06 23:52:48,143 INFO    MainThread:38942 [wandb_init.py:_log_setup():357] Logging user logs to /Users/kiranganeshan/reading-group/RL/wandb/run-20211006_235248-24k4fncv/logs/debug.log
2021-10-06 23:52:48,143 INFO    MainThread:38942 [wandb_init.py:_log_setup():358] Logging internal logs to /Users/kiranganeshan/reading-group/RL/wandb/run-20211006_235248-24k4fncv/logs/debug-internal.log
2021-10-06 23:52:48,143 INFO    MainThread:38942 [wandb_init.py:init():390] calling init triggers
2021-10-06 23:52:48,143 INFO    MainThread:38942 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-10-06 23:52:48,144 INFO    MainThread:38942 [wandb_init.py:init():435] starting backend
2021-10-06 23:52:48,144 INFO    MainThread:38942 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-06 23:52:48,153 INFO    MainThread:38942 [backend.py:ensure_launched():135] starting backend process...
2021-10-06 23:52:48,160 INFO    MainThread:38942 [backend.py:ensure_launched():140] started backend process with pid: 38968
2021-10-06 23:52:48,161 INFO    MainThread:38942 [wandb_init.py:init():440] backend started and connected
2021-10-06 23:52:48,165 INFO    MainThread:38942 [wandb_init.py:init():499] updated telemetry
2021-10-06 23:52:48,166 INFO    MainThread:38942 [wandb_init.py:init():522] communicating current version
2021-10-06 23:52:48,912 INFO    MainThread:38942 [wandb_init.py:init():527] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-06 23:52:48,912 INFO    MainThread:38942 [wandb_init.py:init():535] communicating run to backend with 30 second timeout
2021-10-06 23:52:49,061 INFO    MainThread:38942 [wandb_init.py:init():562] starting run threads in backend
2021-10-06 23:52:51,458 INFO    MainThread:38942 [wandb_run.py:_console_start():1651] atexit reg
2021-10-06 23:52:51,459 INFO    MainThread:38942 [wandb_run.py:_redirect():1525] redirect: SettingsConsole.REDIRECT
2021-10-06 23:52:51,459 INFO    MainThread:38942 [wandb_run.py:_redirect():1530] Redirecting console.
2021-10-06 23:52:51,461 INFO    MainThread:38942 [wandb_run.py:_redirect():1586] Redirects installed.
2021-10-06 23:52:51,461 INFO    MainThread:38942 [wandb_init.py:init():587] run started, returning control to user process
2021-10-06 23:52:51,480 INFO    MainThread:38942 [wandb_watch.py:watch():43] Watching
2021-10-06 23:52:51,506 INFO    MainThread:38942 [wandb_run.py:_atexit_cleanup():1621] got exitcode: 1
2021-10-06 23:52:51,507 INFO    MainThread:38942 [wandb_run.py:_restore():1593] restore
2021-10-06 23:52:54,492 INFO    MainThread:38942 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 689
}

2021-10-06 23:52:54,636 INFO    MainThread:38942 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 689
}

2021-10-06 23:52:55,109 INFO    MainThread:38942 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 10571
}

2021-10-06 23:52:55,212 INFO    MainThread:38942 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 10571
}

2021-10-06 23:52:55,317 INFO    MainThread:38942 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 10571
}

2021-10-06 23:52:55,421 INFO    MainThread:38942 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}

2021-10-06 23:52:55,528 INFO    MainThread:38942 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}

2021-10-06 23:52:55,635 INFO    MainThread:38942 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}

2021-10-06 23:52:55,743 INFO    MainThread:38942 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}

2021-10-06 23:52:55,861 INFO    MainThread:38942 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}

2021-10-06 23:52:56,168 INFO    MainThread:38942 [wandb_run.py:_wait_for_finish():1746] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}
local_info {
}

2021-10-06 23:52:57,282 INFO    MainThread:38942 [wandb_run.py:_append_files():2009] logging synced files
